modules/33acrossAnalyticsAdapter.js

```diff
@@ -235,7 +237,25 @@ function enableAnalyticsWrapper(config) {
     auctions: {},
   };
 
+  if (isGptPubadsDefined()) {
```

Comment:
is this conditional statement strictly needed?

Reply:
removed